src/immutable_data.rs

```diff
@@ -17,12 +17,14 @@ use std::{
 };
 use tiny_keccak;
 
-/// Maximum allowed size for a serialised Immutable Data (ID) to grow to
+/// Maximum allowed size for a serialised ImmutableData to grow to.
 pub const MAX_IMMUTABLE_DATA_SIZE_IN_BYTES: u64 = 1024 * 1024 + 10 * 1024;
 
+/// Unpublished ImmutableData: an immutable chunk of data which can be deleted.
```

Comment:
Probably worth adding that it also can't be fetched by anyone but the listed `owner`.